SpaceCore/SpaceCore.cs

```diff
@@ -20,6 +20,7 @@ internal class SpaceCore : Mod
     {
         public Configuration Config { get; set; }
         internal static SpaceCore Instance;
+        internal static IReflectionHelper Reflection;
```

Comment:
Added this to again reduce indirection to help make funcs that are frequently called to go faster.